### ISSUE TITLE
Heading not showing on localhost when wrapped with Tailwind component

### ISSUE THREAD

**Issue by devklepacki** [resend/react-email#1420]
### Describe the Bug

`Heading` component is not rendering when it's inside a `Tailwind` component. I was building a new email and was fighting with `Heading` not displaying (not rendering in DOM at all). Once I've removed the `Tailwind` component wrapping my template the `Heading` appeared. The same happens with newly created project using `yarn create email`.

- There is no `Join Enigma on Vercel` heading visible right under the logo

![Screenshot 2024-04-20 at 11 29 00](https://github.com/resend/react-email/assets/20723654/970d07b0-5074-4b2a-8258-dd2aae5964d9)

- While it's in the React code:

![Screenshot 2024-04-20 at 11 28 49](https://github.com/resend/react-email/assets/20723654/0ef9c1f6-b548-4cca-87cc-5e277df973de)

- It's also not in the rendered HTML code snippet:

```html
<table align="center" width="100%" border="0" cellPadding="0" cellSpacing="0" role="presentation" style="margin-top:32px">
  <tbody>
    <tr>
      <td><img alt="Vercel" height="37" src="/static/vercel-logo.png" style="display:block;outline:none;border:none;text-decoration:none;margin-top:0px;margin-bottom:0px;margin-left:auto;margin-right:auto" width="40" /></td>
    </tr>
  </tbody>
</table>
<p style="font-size:14px;line-height:24px;margin:16px 0;color:rgb(0,0,0)">Hello <!-- -->alanturing<!-- -->,</p>
<p style="font-size:14px;line-height:24px;margin:16px 0;color:rgb(0,0,0)"><strong>Alan</strong> (<a href="mailto:alan.turing@example.com" style="color:rgb(37,99,235);text-decoration:none;text-decoration-line:none" target="_blank">alan.turing@example.com</a>) has invited you to the <strong>Enigma</strong> team on<!-- --> <strong>Vercel</strong>.</p>
```

- Once the `Tailwind` component is removed, the `Heading` appears

![Screenshot 2024-04-20 at 11 30 56](https://github.com/resend/react-email/assets/20723654/02da5bb8-aa2a-4e33-82c4-36f74f3e3c9c)

### Which package is affected (leave empty if unsure)

@react-email/heading, @react-email/tailwind

### Link to the code that reproduces this issue

https://github.com/devklepacki/react-email-reproduction

### To Reproduce

- Download the project (from reproduction or with `yarn create email`, install dependencies and run it
- On localhost go to _vercel-invite-user_ email template
- There is no _Join Enigma on Vercel_ `Heading` visible right under the logo

### Expected Behavior

Heading contents should be shown when used inside Tailwind component

### What's your node version? (if relevant)

v20.11.0

**Comment by fominv** [comment]
Have the same issue. Downgrading to `"@react-email/components": "0.0.15"` fixed it for me.

**Comment by peperoli** [comment]
> Have the same issue. Downgrading to `"@react-email/components": "0.0.15"` fixed it for me.

Same here, but had the issue with `"@react-email/components": "0.0.15"` too. On `0.0.14` it's working for me though.

**Comment by gabrielmfern** [comment]
This should be fixed under the latest canaries, for both `@react-email/components` and `@react-email/tailwind`. No need to downgrade here.

See https://github.com/resend/react-email/issues/1391#issuecomment-2035304838.

**Comment by poeck** [comment]
For me `@react-email/tailwind@0.0.16-canary.1` worked, but `@react-email/tailwind@0.0.16` didn't.

**Comment by gabrielmfern** [comment]
@poeck Yeah, I had to compromise there since we needed a new stable release urgently to fix the issues with peer dependencies, so the changes from canary weren't published. I should tune these versions out properly in the coming weeks. Probably making a new latest release with those.

**Comment by mfts** [comment]
I'm on `@react-email/components` and the `<Heading>` is completely missing from development and production emails. 

Not sure how long this has been going on but it's super weird. 

I'm changing `<Heading>` to `<Text>` and it will show. That's my short term fix.

**Comment by devklepacki** [comment]
@mfts For me upgrading `tailwind` package to canary release:
```json
"@react-email/tailwind": "0.0.16-canary.1",
```
resolved the issue. Also remember to import `Tailwind` component separately `from @react-email/tailwind` instead of `/components`

**Comment by gabrielmfern** [comment]
Released a new canary for `@react-email/tailwind` that should be working with this. It is also on the latest canary for `@react-email/components`.

**Comment by andresgutgon** [comment]
Can confirm. I was having exactly this issue and these versions fix it

``` json
  "dependencies": {
    "react": "^18.2.0",
    "@react-email/components": "0.0.18-canary.0",
    "@react-email/tailwind": "0.0.17-canary.0",
    "react-email": "2.1.3-canary.2"
  },
```

**Comment by gabrielmfern** [comment]
This has been fixed and released under [`@react-email/tailwind@0.0.17`](https://github.com/resend/react-email/releases/tag/%40react-email%2Ftailwind%400.0.17) or `@react-email/components@0.0.18`. If anyone still experiences this, please let me know and I'll reopen the issue.

**Comment by eltongonc** [comment]
Installing `@react-email/components` and `@react-email/tailwind` separately worked for me
